dietpi/dietpi-software

```diff
@@ -13658,6 +13735,20 @@ _EOF_
 
 		fi
 
+		software_id=200 # DietPi-Dashboard
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
+
+			Banner_Uninstalling
+			[[ -d '/opt/dietpi-dashboard' ]] && G_EXEC rm -R /opt/dietpi-dashboard
+			if [[ -f '/etc/systemd/system/dietpi-dashboard.service' ]]; then
+
+				G_EXEC systemctl disable --now dietpi-dashboard
+				G_EXEC rm /etc/systemd/system/dietpi-dashboard.service
+
+			fi
```

Comment:
In case `dietpi-services` was used to change process priorities, or a manual override was created:
```sh
[[ -d '/etc/systemd/system/dietpi-dashboard.service.d' ]] && G_EXEC rm -R /etc/systemd/system/dietpi-dashboard.service.d
```
And I personally prefer the service to be disabled/removed first and the binary it uses afterwards. It doesn't make a difference here, but is the better convention to cover cases where the service contains an `ExecStop`, which would then fail.